scripts/demoDeBridge.ts

```diff
@@ -0,0 +1,179 @@
+import { providers, Wallet, utils, constants, Contract } from 'ethers'
```

Comment:
same as with deploy script. Not sure if we still need .ts files.

Reply:
it's a demo test script